src/components/common/Dialogs/ControlSafeDialog/TransactionTypesSection/RawTransactionSection.tsx

```diff
@@ -49,11 +49,6 @@ const RawTransactionSection = ({
         labelValues={{
           span: labelValues,
         }}
-        formattingOptions={{
-          numeral: true,
-          numeralPositiveOnly: true,
-          numeralDecimalScale: 0,
-        }}
```

Comment:
why is this removed?

Reply:
Providing a `formattingOptions` prop to the Input component makes the component use `cleave.js` and unfortunately, `cleave.js` which is the library that we use to format the values of numeral inputs, has a bug in which it doesn't update itself after a re-render.

So in context what would happen is this:

- You want to make a multi-transaction, you add a Transfer NFT and a Transfer Funds transactions.
- You decide that you don't want to transfer the NFT so you remove the tab from the dialog.
- Now the Transfer Funds goes from having a `transactionIndex` of 1 to 0.
- This change of state will not make `cleave.js` automatically update the input in the re-render, so if it had a `transactionIndex` of 1 before, and you interact with it after removing the tab, it will behave as if the `transactionIndex` was the old one, and that causes more problems like the validation breaking because it thinks that there's still 2 transactions in the form when there's only 1.


Unluckily, `Cleave.js` is no longer maintained so to solve this we would need to either find some hacky way of making it work or outright looking for another library and refactoring the Input component